en/identity-server/next/docs/guides/login/cross-protocol-logout.md

```diff
@@ -0,0 +1,127 @@
+# Configure Cross-Protocol Logout
+
+This page guides you through enabling cross-protocol logout across different protocols such as SAML and OpenID Connect in WSO2 Identity Server.
+
+## Register the service providers
+
+You need to register the two sample applications as service providers in WSO2 Identity Server.
+
+### Pickup Dispatch (SAML)
+
+Follow the steps given below to register a SAML application as a service provider in WSO2 IS.
+
+1.  Log in to the management console (`https://<IS_HOST>:<PORT>/carbon`) using admin/admin credentials. 
```

Comment:
As per the linked github issue, this documentation is for IS 7. In that case, shouldn't we refer to the new React console in documentation rather than mgt console, since mgt console is planned to be deprecated from IS 7?